tests/lib/util/glob-util.js

```diff
@@ -275,19 +267,45 @@ describe("globUtil", () => {
         });
 
         it("should not return a file which does not exist", () => {
-            const patterns = ["tests/fixtures/glob-util/hidden/bar.js"];
-            const result = globUtil.listFilesToProcess(patterns);
+            const filename = getFixturePath("glob-util", "hidden", "bar.js");
+            const patterns = [filename];
+
+            assert.throws(() => {
+                globUtil.listFilesToProcess(patterns, {
+                    cwd: getFixturePath(),
+                    allowMissingGlobs: true
+                });
+            }, `'${filename}' was not found.`);
+        });
+
+        it("should throw if a folder that does not have any applicable files is linted", () => {
+            const filename = getFixturePath("glob-util", "empty");
+            const patterns = [filename];
 
-            assert.strictEqual(result.length, 0);
+            assert.throws(() => {
+                globUtil.listFilesToProcess(patterns, {
+                    cwd: getFixturePath()
+                });
+            }, `'${filename}' was not found.`);
+        });
+
+        it("should throw if only ignored files match a glob", () => {
+            const pattern = getFixturePath("glob-util", "ignored");
+            const options = { ignore: true, ignorePath: getFixturePath("glob-util", "ignored", ".eslintignore") };
+
+            assert.throws(() => {
+                globUtil.listFilesToProcess([pattern], options);
+            }, `All files matched by '${pattern}' are ignored.`);
         });
 
         it("should not return an ignored file", () => {
```

Comment:
Is this test name still accurate given that we have an `assert.throws` in the test body?